server/form-pages/apply/basic-information/oralHearing.ts

```diff
@@ -29,6 +31,18 @@ export default class OralHearing implements TasklistPage {
     return 'release-date'
   }
 
+  response() {
+    const response = {
+      [this.title]: convertToTitleCase(this.body.knowOralHearingDate),
+    } as Record<string, string>
```

Comment:
I'm not at all sure how these types work but I wonder if there would be any merit in making it a bit clearer that these `Record` objects represent individual pairings of a question with its answer e.g. with `Record<Question, Answer>` ?